app/controllers/api/red_hat_migration_analytics_controller.rb

```diff
@@ -46,11 +63,23 @@ def find_provider_ids(type)
     end
 
     class << self
-      def parse_manifest
-        default_manifest_path = Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")
-        user_manifest_path    = Pathname.new("/opt/rh/cfme-migration_analytics/manifest.json")
-        manifest_path         = user_manifest_path.exist? ? user_manifest_path : default_manifest_path
 
+      def default_manifest_path
+        @default_manifest_path ||= Cfme::MigrationAnalytics::Engine.root.join("config", "default-manifest.json")
+      end
+
+      def user_manifest_path
+        @user_manifest_path ||= Pathname.new("/opt/rh/cfme-migration_analytics/manifest.json")
+      end
+
+      def store_manifest(manifest)
+        File.open(user_manifest_path, 'w') do |f|
+          f.write(JSON.generate(manifest))
+        end
+      end
+
+      def parse_manifest
+        manifest_path = user_manifest_path.exist? ? user_manifest_path : default_manifest_path
         manifest = Vmdb::Settings.filter_passwords!(load_manifest(manifest_path))
 
         {
```

Comment:
Discussing with @vconzola, we want to be able to display the version of the default manifest even if there is a user manifest present (to explain what version will be in use after `reset_manifest`). Would it be possible to change the logic of `parse_manifest` so it loads both manifest files, and has an additional `default_version` field? And then in the `index` method we can add `:default_manifest_version => manifest[:default_version]`.

I figured it might make sense to do this while you're in here, if you prefer I can make my own PR for this.